backend/de.metas.marketing/base/src/main/java/de/metas/marketing/base/interceptor/MKTG_Channel.java

```diff
@@ -56,18 +56,18 @@ public void checkIfCanBeDeleted(@NonNull final I_MKTG_Channel mktgChannel)
 	{
 		boolean canBeDeleted = true;
 		Set<UserId> usersSet = mktgChannelDao.retrieveUsersHavingChannel(mktgChannel.getMKTG_Channel_ID());
-		for (UserId userId : usersSet)
+		for (final UserId userId : usersSet)
 		{
-			if (mktgChannelDao.retrieveMarketingChannelsCountForUser(userId) == 1)
+
+			if (mktgChannelDao.retrieveMarketingChannelsCountForUser(userId) > 0 && !userDAO.isSystemUser(userId))
```

Comment:
Even for system-users, you won't be able to delete the channel, simply because of the foreign-key-constraint we talked about.
Remember, you tried it when we had the screen-session..

Reply:
Yeah, forgot about that 😓 Removed the condition.